app/tests/algorithms_tests/test_tasks.py

```diff
@@ -315,7 +302,7 @@ def test_algorithm(
     )
     heatmap_interface = ComponentInterface.objects.get(slug="generic-overlay")
     interface = AlgorithmInterfaceFactory(
-        inputs=[input_interface],
+        inputs=[input_interface, heatmap_interface],
         outputs=[json_result_interface, heatmap_interface],
     )
```

Comment:
Why this change?

Reply:
The new executor just copies the io. The test used a different container that was creating a bunch of stuff, including the deprecated overlay interface. The test does too much. This allows minimal changes to the test by letting the new executor just copy the io.